UPDATING.md

```diff
@@ -1047,6 +1047,24 @@ If the DAG relies on tasks with other trigger rules (i.e. `all_done`) being skip
 
 The goal of this change is to achieve a more consistent and configurale cascading behaviour based on the `BaseBranchOperator` (see [AIRFLOW-2923](https://jira.apache.org/jira/browse/AIRFLOW-2923) and [AIRFLOW-1784](https://jira.apache.org/jira/browse/AIRFLOW-1784)).
 
+
+### RedshiftToS3Transfer:: signature changed
+
+Previous versions of the `RedshiftToS3Transfer` operator required `schema` and `table` arguments as the first 2
```

Comment:
The order of the arguments does not matter in operators. Each operator must use named arguments, because we have ``apply_default`` decorator.

Reply:
I see! So does this mean that this is not a breaking change @mik-laj ?

Reply:
Yes. It is not breaking changes.

Reply:
Ah! understood. So I am assuming that there is no need to document this in UPDATING.md. @mik-laj

Reply:
It does not require a note in UPDATING.md.